src/crs/projected/eqareaalbers.jl

```diff
@@ -0,0 +1,153 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Albers(lat₀, lat₁, lat₂, Datum, Shift)
+    Albers CRS with latitude origin lat₀ standard parallels `lat₁` and `lat₂`, `lon₀`, `Datum` and `Shift`.
+
+## Examples
+
+```julia
+Albers(1, 1) # add default units
+Albers(1m, 1m) # integers are converted converted to floats
+Albers(1.0km, 1.0km) # length quantities are converted to meters
+Albers(1.0m, 1.0m)
+Albers{NAD83}(1.0m, 1.0m)
+```
+
+See [EPSG:5070](https://epsg.io/5070).
+"""
+struct Albers{lat₀,lat₁,lat₂,lon₀,Datum,Shift,M<:Met} <: Projected{Datum,Shift}
```

Comment:
```suggestion
struct Albers{latₒ,lat₁,lat₂,Datum,Shift,M<:Met} <: Projected{Datum,Shift}
```
Please remove "longitude origin" from the projection parameters as it is part of the "shift parameters".
Rename "latitude origin" from `lat\_0` to `lat\_o`.